sdk/tools/mkshelllink/mkshelllink.c

```diff
@@ -195,6 +210,13 @@ int main(int argc, const char *argv[])
         return 0;
     }
 
+    // The icon of text files
+    if (strstr(pszIcon, ".txt"))
```

Comment:
This should be good enough.
Possible improvements, I guess:
compare to end of string only,
be case-insensitive.

Reply:
Improved.